src/modules/associated-test-modal/associated-test-modal.tsx

```diff
@@ -95,7 +90,14 @@ export const AssociatedTestModal = () => {
               </div>
             )}
             gridTemplateColumns="400px 456px auto"
-            data={tests}
+            data={tests.map((test) => ({
```

Comment:
mb improve testsTransform ?

Reply:
so testsTransform will be very difficult